src/renderer/modals/AddAccounts/index.js

```diff
@@ -243,6 +253,10 @@ class AddAccounts extends PureComponent<Props, State> {
 
     const errorSteps = err ? [2] : [];
 
+    if (stepId === "chooseCurrency" && this.props.skipChooseCurrencyStep) {
```

Comment:
Why would we reach this case? Aren't we shifting the steps when we have the `skipChooseCurrencyStep` flag in the first place?

Reply:
yes, we reach this case. 
In fact, we have this: https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/renderer/modals/AddAccounts/index.js#L126
So the stepId is set to "chooseCurrency" initially. 
I can either set the initial stepid = connectDevice (if skipChooseCurrencyStep == true)
or do the way what I did.